pkgs/development/tools/language-servers/nixd/default.nix

```diff
@@ -81,6 +81,7 @@ stdenv.mkDerivation rec {
   meta = {
     description = "Nix language server";
     homepage = "https://github.com/nix-community/nixd";
+    changelog = "https://github.com/nix-community/nixd/releases/tag/${version}";
     license = lib.licenses.lgpl3Plus;
     maintainers = with lib.maintainers; [ inclyc Ruixi-rebirth ];
```

Comment:
LGTM, and, would you like to add yourself in this list?

Reply:
Sure, I added myself to maintainers :)